content/develop/api-reference/charts/altair_chart.md

```diff
@@ -11,7 +11,7 @@ keywords: altair_chart, altair, chart, visualization, data, plot, graph, vega-li
 
 <Autofunction function="VegaLiteState" />
 
-<Autofunction function="DeltaGenerator.add_rows" />
+<Autofunction function="DeltaGenerator.add_rows" deprecated={true} deprecatedText="<code>.add_rows()</code> was deprecated in version 1.52.0 and will be removed in a later version." />
```

Comment:
nit: Can we add the link to the feedback issue somewhere: https://github.com/streamlit/streamlit/issues/13063? For now this is a bit more of a soft deprecation (only showing a log and nothing on UI) with the main goal of getting feedback from users: